### PR TITLE
EEC-4

### PR DESCRIPTION
**Pull request by jamiecarterHO**
## What?

Added 'How should we contact you about your eVisa?' and 'Are you completing this form on behalf of someone else?' question pages.

## Why?

[EEC-4](https://collaboration.homeoffice.gov.uk/jira/browse/EEC-4)

## How?

Added step and field definitions, updated or added translations and validations

## Anything Else? (optional)

Updated field names to use 'requestor' instead of 'reporter' - this is a term used in body copy of the form. This feels more appropriate.

## Check list

- [x] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: apps/eec/fields/index.js [hunk 1]
```diff
@@ -124,89 +124,187 @@ module.exports = {
...
+      field: 'requestor-contact-method',
+      value: 'uk-address'
+    }
+  },
+  'requestor-county': {
+    className: ['govuk-input', 'govuk-!-width-two-thirds'],
+    dependent: {
+      field: 'requestor-contact-method',
+      value: 'uk-address'
+    }
```
Comment (dk4g) [inline apps/eec/fields/index.js:264]: this can be removed as there is no validation for this field

File: apps/eec/fields/index.js [hunk 2]
```diff
@@ -124,89 +124,187 @@ module.exports = {
...
+  },
+  'requestor-address-line-2': {
+    dependent: {
+      field: 'requestor-contact-method',
+      value: 'uk-address'
+    }
```
Comment (dk4g) [inline apps/eec/fields/index.js:250]: this also can be removed

File: apps/eec/translations/src/en/pages.json
Comment (dk4g) [inline apps/eec/translations/src/en/pages.json:1]: someone-else-details page will require custom template to display `consent` paragraph

**Review by dk4g — CHANGES_REQUESTED** [review]

File: apps/eec/fields/index.js (hunk 1 above)
Comment (jamiecarterHO) [reply apps/eec/fields/index.js]: removed

File: apps/eec/fields/index.js (hunk 2 above)
Comment (jamiecarterHO) [reply apps/eec/fields/index.js]: removed

File: apps/eec/translations/src/en/pages.json
Comment (jamiecarterHO) [reply apps/eec/translations/src/en/pages.json:1]: Added

**Review by dk4g — APPROVED** [review]

**Pull request merged**